### ISSUE TITLE
SVN 1.7 support

### ISSUE THREAD

**Issue by GoogleCodeExporter** [salvatorelai/svnx#161]
```
What steps will reproduce the problem?

1. install svn 1.7 on your system
2. manually checkout any repository
3. open the working space with svnx

What is the expected output? What do you see instead?

svnx should work, instead it interpret 1.7 as an old incompatible version

What version of the product are you using? On what operating system?

svnX 1.3.2 on OSX Snow Leopard, svn 1.7 from Macports

```

Original issue reported on code.google.com by `devis.lu...@yahoo.com` on 1 Mar 2012 at 2:19
- Merged into: #159

**Comment by GoogleCodeExporter** [comment]
Original comment by `chris...@gmail.com` on 2 Mar 2012 at 12:45
- Changed state: **Duplicate**

**Comment by GoogleCodeExporter** [comment]
```
Fixed as suggested in #159, reconfiguring svn bin path in svnx preferences
```

Original comment by `de...@oracolo.com` on 2 Mar 2012 at 10:28
